pyodide-build/pyodide_build/buildpkg.py

```diff
@@ -49,7 +49,7 @@ def _make_whlfile(*args, owner=None, group=None, **kwargs):
 
 
 class BashRunnerWithSharedEnvironment:
-    """Run multiple bash scripts with persisent environment.
+    """Run multiple bash scripts with persistent environment.
```

Comment:
It would be nice if the bashrunner would print the script it tried to run if the script failed. Currently it appears with a traceback with the bash script collapsed like `cmd1\ncmd2\ncmd3` which is pretty hard to read.

Reply:
Okay, will look at this!

Reply:
Not necessary in this PR if you don't feel like it, but it would be helpful.

Reply:
We could also really use to fix the bashrunner so that it returns an error if the script has an error rather than hanging or returning 0.

Reply:
This also needs some rewriting already (see #2194), maybe could do both.